system_files/desktop/shared/usr/share/ublue-os/just/82-bazzite-apps.just

```diff
@@ -157,6 +157,48 @@ install-opentabletdriver:
     systemctl enable --user --now arch-opentabletdriver.service && \
     distrobox enter -n arch -- bash -c 'distrobox-export --app otd-gui'
 
+# Install/update Davinci Resolve, a closed-source video editing utility
+install-resolve:
+    #!/usr/bin/bash
+    source /usr/lib/ujust/ujust.sh
+    set -eo pipefail
+    # Remove the existing container
+    if podman container inspect davincibox >/dev/null 2>&1; then 
+      echo "Removing existing davincibox container"
+      podman stop --ignore davincibox && podman rm davincibox
+    fi
+    DOWNLOADDIR=$(xdg-user-dir DOWNLOAD || echo ${HOME})
+    echo "${bold}This script requires you to download DaVinci Resolve installer manually from their website and place it in ${HOME} or ${DOWNLOADDIR}${n}"
+    echo "${bold}https://www.blackmagicdesign.com/event/davinciresolvedownload${n}"
+    # Download davincibox setup script
+    SETUPSCRIPT=/tmp/davincibox_setup.sh
+    wget -O $SETUPSCRIPT https://raw.githubusercontent.com/zelikos/davincibox/main/setup.sh
```

Comment:
From a supply-chain point of view, it would be good if this script was pinned to a specific commit since it's not managed by ublue